diagnostic.md

```diff
@@ -44,9 +56,23 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
+```CSS
+.big {
+
+}
+#contentPane {
+
+}
+.important .red {
```

Comment:
This will select elements that have ```red``` as a class, that is inside of an element that has ```important``` as a class.  To select elements that have both ```important``` and ```red```, use ```.important.red```.